ghost/core/core/frontend/helpers/tpl/content-cta.hbs

```diff
@@ -2,13 +2,13 @@
 <aside class="gh-post-upgrade-cta">
     <div class="gh-post-upgrade-cta-content" style="background-color: {{accentColor}}">
```

Comment:
Not sure when this happened, but IMO this should always be `{{@site.accentColor}}`. It's a global property, there's no reason to extend the theme API here and a local copy?

Reply:
Will still keep the local copy of `accentColor` property for themes as otherwise it will be a breaking change for theme devs already relying on that in custom cta helpers, and we can get rid of that in 5.0.